src/components/DevTools/DevTools.anticov.ts

```diff
@@ -1,7 +1,17 @@
-import * as sdk from '@kiltprotocol/sdk-js'
+import Kilt, {
+  DelegationRootNode,
+  Identity,
+  BlockchainUtils,
+  DelegationNode,
+  MessageBody,
+  MessageBodyType,
+  Permission,
+  UUID,
+} from '@kiltprotocol/sdk-js'
 import { IMetadata } from '@kiltprotocol/sdk-js/build/types/CTypeMetadata'
 import { ICTypeSchema } from '@kiltprotocol/sdk-js/build/types/CType'
 import BN from 'bn.js'
+import { IS_IN_BLOCK } from '@kiltprotocol/sdk-js/build/blockchain/Blockchain.utils'
```

Comment:
You can just use `BlockchainUtils.IS_IN_BLOCK`, or if you want to have `IS_IN_BLOCK` as separate variable, write something like this after the import section:
```
const { IS_IN_BLOCK } = BlockchainUtils
```